mrp_unbuild_cust_alu_analytics/views/mrp_unbuild_incidence_views.xml

```diff
@@ -25,16 +26,19 @@
             <search>
                 <field name="incidence_type_id"/>
                 <field name="detail"/>
+                <field name="start_incidence"/> 
```

Comment:
What effect does searching for a Date field in a search view have?

Reply:
We're all agree with this: search field should be removed